pkg/app/config.go

```diff
@@ -0,0 +1,99 @@
+package app
+
+import (
+	"github.com/pkg/errors"
+	kotsv1beta1 "github.com/replicatedhq/kots/kotskinds/apis/kots/v1beta1"
+	kotsconfig "github.com/replicatedhq/kots/pkg/config"
+	"github.com/replicatedhq/kots/pkg/crypto"
+	"github.com/replicatedhq/kots/pkg/template"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+func templateConfig(config *kotsv1beta1.Config, configValues *kotsv1beta1.ConfigValues, license *kotsv1beta1.License, encryptionKey string) (string, error) {
```

Comment:
This looks like something that should belong in kots, not kotsadm. Is there a function in kots that we can vendor in to provide this functionality?